README.md

```diff
@@ -8,4 +8,5 @@ To get started:
 - Clone this repo: `cd; git clone https://github.com/jonathan-backer/sps-2021.git`
 - Run the server:
   - `cd ~/sps-2021/portfolio`
```

Comment:
Perhaps in a follow up PR you could modify the instructions in the README.md  to reference the new team_project directory.